tests/performance/timing_step_37.cc

```diff
@@ -412,6 +413,35 @@ LaplaceProblem<dim>::setup_dofs()
   VectorTools::interpolate_boundary_values(
     mapping, dof_handler, 0, Functions::ZeroFunction<dim>(), constraints);
   constraints.close();
+
+  // Renumber DoFs
+  typename MatrixFree<dim, float>::AdditionalData additional_data;
+  additional_data.tasks_parallel_scheme =
+    MatrixFree<dim, float>::AdditionalData::none;
+
+  std::set<types::boundary_id> dirichlet_boundary;
+  dirichlet_boundary.insert(0);
```

Comment:
You can write these things simpler these days:
```suggestion
  const std::set<types::boundary_id> dirichlet_boundary = {0};
```
But it doesn't really matter.